test/models/RegularPackageSpec.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import generators.ModelGenerators
+import org.scalacheck.Gen
+import org.scalatest.{FreeSpec, MustMatchers}
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
+import play.api.libs.json.{JsError, JsSuccess, Json}
+
+class RegularPackageSpec extends FreeSpec with MustMatchers with ScalaCheckPropertyChecks with ModelGenerators {
+
+  "Regular Package" - {
+
+    "must deserialise when the kind of package does not indicate Bulk or Unpacked" in {
+
+      val gen = for {
+        kindOfPackage    <- stringWithMaxLength(3)
```

Comment:
Similar question here as above

Reply:
Same as above, this is for a `RegularPackage`